exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/ApiController.java

```diff
@@ -295,4 +402,17 @@ private void failureHandler(RoutingContext rc) {
     // This throwable must correspond to an internal server error.
     return Optional.empty();
   }
+
+  @VisibleForTesting
+  static Map<HashCode, String> hexEncodeTransactionMessages(
+      Map<HashCode, TransactionMessage> txMessages) {
+    return txMessages.entrySet()
+        .stream()
+        .collect(Collectors.toMap(Map.Entry::getKey,
```

Comment:
static import

Reply:
May do for both `toMap` and `Entry`: `.collect(toMap(Entry::getKey,`